__tests__/app.test.js

```diff
@@ -1,157 +1,160 @@
+    .get("/api/reviews/not-a-number")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Invalid input");
     });
 });
 
-describe('GET /api/reviews/:review_id/comments', () => {
-    test('200: responds with an array of comments for the given review_id', () => {
-        return request(app)
-        .get('/api/reviews/2/comments')
-        .expect(200)
-        .then(({ body }) => {
-            const { comments } = body;
-            expect(Array.isArray(comments)).toBe(true);
-            comments.forEach((comment) => {
-                expect(comment).toHaveProperty('comment_id', expect.any(Number));
-                expect(comment).toHaveProperty('votes', expect.any(Number));
-                expect(comment).toHaveProperty('created_at', expect.any(String));
-                expect(comment).toHaveProperty('author', expect.any(String));
-                expect(comment).toHaveProperty('body', expect.any(String));
-                expect(comment).toHaveProperty('review_id', 2);
-            });
-        });
-    });
-    test('200: responds with a message if there are no comments for the given review_id', () => {
-        return request(app)
-        .get('/api/reviews/1/comments')
-        .expect(200)
-        .then(({ body }) => {
-            const { comments } = body;
-            expect(comments).toEqual([]);
-        });
-    });
-    test('404: responds with a message if review_id is not found', () => {
-        return request(app)
-        .get('/api/reviews/999/comments')
-        .expect(404)
-        .then(({ body }) => {
-            expect(body.msg).toBe('Review not found!');
-        });
-    });
-    test('400: responds with a message if review_id is not a number', () => {
-        return request(app)
-        .get('/api/reviews/not-a-number/comments')
-        .expect(400)
-        .then(({ body }) => {
-            expect(body.msg).toBe('Invalid input');
-        });
-    });
-    test('responds with an array of comments for the given review_id in most recent order', () => {
-        return request(app)
-        .get('/api/reviews/2/comments')
-        .expect(200)
-        .then(({ body }) => {
-            const { comments } = body;
-            expect(comments).toBeSortedBy('created_at', {descending: true});
-        });
-    });
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("returns the posted comment with status 201", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "bainesface", body: "TEST COMMENT!" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.body).toBe("TEST COMMENT!");
+      });
+  });
+  test("post 201, ignores any unnecessary content", () => {
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send({ username: "bainesface", body: "TEST COMMENT!", fruit: "apples" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.body).toBe("TEST COMMENT!");
```

Comment:
These happy tests are all great, but i would expect our comments to come back on a key of comments so
expect(body.comments.body).toBe("TEST COMMENT!")
Only need to change the 2x happy path tests and the send the response back on a key of comments to fix